Makefile

```diff
@@ -347,7 +347,8 @@ check-contrib:
 	@echo Setting contrib at $(CONTRIB_PATH) to use this core checkout
 	make -C $(CONTRIB_PATH) for-all CMD="go mod edit -replace go.opentelemetry.io/collector=$(CURDIR)"
 	make -C $(CONTRIB_PATH) for-all CMD="go mod edit -replace go.opentelemetry.io/collector/model=$(CURDIR)/model"
-	make -C $(CONTRIB_PATH) for-all CMD="go mod tidy"
```

Comment:
is this to ensure compatibility with both?

Reply:
yes.